tests/_helm/values/nightly/standalone

```diff
@@ -27,6 +27,7 @@ image:
     repository: harbor.milvus.io/milvus/milvus
     tag: nightly-20240821-ed4eaff
 indexCoordinator:
+  enabled: false
```

Comment:
remember to add enabled: false to indexNode:

Reply:
done